episodes/2-keras.Rmd

```diff
@@ -211,15 +211,6 @@ determine the relevant inputs and outputs.
 ## 3. Prepare data
 The input data and target data are not yet in a format that is suitable to use for training a neural network.
 
-### Change types if needed
-First, the species column is our categorical target, however pandas still sees it as the
-generic type `Object`. We can convert this to the pandas categorical type:
-```python
-penguins['species'] = penguins['species'].astype('category')
-```
-
-This will make later interaction with this column a little easier.
-
```

Comment:
I tested and this does not make any difference for the rest of the episode... Maybe this was necessary in an old edition of pandas?